lxd/instance/instance_interface.go

```diff
@@ -241,4 +241,6 @@ type MigrateReceiveArgs struct {
 
 	InstanceOperation *operationlock.InstanceOperation
 	Refresh           bool
+	Convert           bool
```

Comment:
Is `Convert` used?

Reply:
Thanks. This was a left-over from initial implementation.